scripts/tasks/src/verify-packaging.ts

```diff
@@ -41,13 +41,15 @@ export function verifyPackaging(options: Options) {
 
   const processedResult = npmPackResult.output
     .toString()
-    .replace(/\bnpm notice\b\s+[\d.]+[kB]+\s+/gi, '')
+    .replace(/\bnpm notice\b\s+[\d.]+[MkB]+\s+/gi, '')
```

Comment:
react bundle has above 1MB of javascript -> using UMD in production should be avoided